CefSharp.MinimalExample.Wpf/CefSharp.MinimalExample.Wpf.csproj

```diff
@@ -67,8 +67,10 @@
     <ApplicationManifest>app.manifest</ApplicationManifest>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="Microsoft.Expression.Interactions, Version=4.5.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL" />
```

Comment:
>Project 'CefSharp.Core' could not be loaded because it's missing install components. To fix this launch Visual Studio setup with the following selections:
Install Visual C++ 2015 Tools for Windows Desktop

That's weird, I do program with C++ here...but I will install those then.